src/Nox.Types/Types/HashedText/TypeOptions/HashedTextTypeOptions.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Nox.Types;
+
+public class HashedTextTypeOptions
```

Comment:
Can we update the Schema for this new TypeOptions?
https://github.com/NoxOrg/Nox.Generator#schemas-update